assets/js/blocks/checkout/inner-blocks/checkout-billing-address-block/block.tsx

```diff
@@ -91,7 +91,14 @@ const Block = ( {
 
 	return (
 		<AddressFormWrapperComponent>
-			<StoreNoticesContainer context={ noticeContexts.BILLING_ADDRESS } />
+			<StoreNoticesContainer
+				context={ noticeContexts.BILLING_ADDRESS }
```

Comment:
I would prefer if we could somehow define an array of context here rather than the separate prop.